torch/utils/hipify/cuda_to_hip_mappings.py

```diff
@@ -7884,6 +7884,37 @@
             ("hipsparseSetMatIndexBase", CONV_MATH_FUNC, API_SPARSE),
         ),
         ("cusparseSetMatType", ("hipsparseSetMatType", CONV_MATH_FUNC, API_SPARSE)),
+        ("cusparseSpMV", ("hipsparseSpMV", CONV_MATH_FUNC, API_SPARSE)),
```

Comment:
Can you push this change in hipify-torch repo also?

Reply:
Done, PR is open there.